package/MDAnalysis/core/groups.py

```diff
@@ -458,8 +550,16 @@ def ix(self):
         return self._ix
 
     @property
+    def ix_array(self):
```

Comment:
`self.ix` has a different meaning in `ComponentBase` and in `GroupBase`. For groups, `self.ix == self.ix_array`; both are the array of indices of the components. For components, instead, `self.ix` is the index of the component as an integer. `self.ix_array` gives a common interface between groups and components to get the index as an array without having to check for the type.

I'll add a word in the doc.